src/printer/index.js

```diff
@@ -56,7 +47,7 @@ function printRubyString(raw, options) {
   // is enclosed with `enclosingQuote`, but it isn't. The string could contain
   // unnecessary escapes (such as in `"\'"`). Always using `makeString` makes
   // sure that we consistently output the minimum amount of escaped quotes.
-  return modifier + util.makeString(rawContent, enclosingQuote);
+  return util.makeString(rawContent, enclosingQuote);
```

Comment:
I believe the `modifier` prefix is specific to Python, i.e. it supports the following syntax:

```python
"hello"
u"hello"
r"hello"
b"hello"
```

We shouldn't need it for ruby 🤞